code/part-two/README.md

```diff
@@ -324,78 +324,49 @@ for a namespace for the transaction family, allowing many families to coexist
-prefix above, and end up with:
-
-```
-5f4d76 00 1b96dbb5322e410816dd41d93571801e751a4f0cc455d8bd58f5f8ad3d67cb
-```
-
-_Note that we have added spaces for readability. Actual addresses have no
-spaces._
+| Namespace (6) | Type prefix (2) | Identifier hash (62)
+| ------------- | --------------- | --------------------
+| `5f4d76`      | `00`            | `1b96dbb5322e410816dd41d93571801e751a4f0cc455d8bd58f5f8ad3d67cb`
 
-#### Collection Prefix
-
-For both cryptomoji and offers (see [extra credit](#extra-credit)), the next
-eight characters are a prefix for the collection they belong to. These are
-generated from the first eight characters of a SHA-512 hash of the collection’s
+A collection will be stored first under a one byte type prefix: `00`. The
+remaining 62 characters are the first 62 characters of a SHA-512 hash of its
 public key.
 
-So, for the same public key above:
+For example, the hash used to create the address above might be generated like
+this:
 
-```
-1b96dbb5
+```javascript
+createHash('sha512').update('034f355bdcb7cc0af728ef3cceb9615d90684bb5b2ca5f859ab0f0b704075871aa').digest('hex');
+// 1b96dbb5322e410816dd41d93571801e751a4f0cc455d8bd58f5f8ad3d67cbee2e9e5c5df5bb65c282f1aaf516cf7cc5a2f7ff592a80cf920e1abaab8d29279f
 ```
 
 #### Cryptomoji
 
-The final 54 characters of a cryptomoji’s address is the first 54 characters of
-a SHA-512 hash of their DNA string. So the address of a cryptomoji with the
-dna `c44b22d96246b0980954f7818567f453039a` would be:
+| Namespace (6) | Type prefix (2) | Collection prefix (8) | Identifier hash (54)
+| ------------- | --------------- | --------------------- | --------------------
+| `5f4d76`      | `01`            | `1b96dbb5`            | `0c8514ab2a7cf361062601716bcd762097e41f9011a5e6f8ff6c5f`
 
-```
-5f4d76 01 1b96dbb5 0c8514ab2a7cf361062601716bcd762097e41f9011a5e6f8ff6c5f
-```
+Cryptomoji need to be divvied up by who owns them. That way you can query a
+partial address to get all of the cryptomoji owned by a public key. So after a
+type prefix of `01` a collection has the first eight characters from a SHA-512
+hash of the owner's public key. The final 54 characters of a cryptomoji’s
+address will the first 54 characters of a SHA-512 hash of their DNA string.
 
 #### Sire Listing
 
-Like collections, the last 62 characters of a sire listing’s address are the
-first 62 characters of a SHA-512 hash of a public key, the sire's owner in this
-case. So our example address would be identical to our example collection, but
-with a different resource prefix:
+| Namespace (6) | Type prefix (2) | Identifier hash (62)
+| ------------- | --------------- | --------------------
+| `5f4d76`      | `02`            | `1b96dbb5322e410816dd41d93571801e751a4f0cc455d8bd58f5f8ad3d67cb`
 
-```
-5f4d76 02 1b96dbb5322e410816dd41d93571801e751a4f0cc455d8bd58f5f8ad3d67cb
-```
+Since collections are only allowed one sire listing, they can be stored under a
```

Comment:
**Clarify pronoun:** Since collections are only allowed one sire listing, ~they~ `sire listings` can be stored under a nearly identical address `format as collections, where the identifier hash is the first 62 characters of a SHA-512 hash of its owner's public key`.